auth-manager/src/main/java/com/qpay/authmanager/service/user/impl/UserServiceImpl.java

```diff
@@ -25,16 +26,18 @@ public class UserServiceImpl implements UserService, UserDetailsService {
 
     private final PasswordEncoder passwordEncoder;
 
-    public UserEntity addUser(final UserModification userModification) {
-        final var newUserEntity = userMapper.map(userModification);
-        newUserEntity.setPassword(passwordEncoder.encode(userModification.password()));
+    public UserEntity addUser(final UserCreation userCreation) {
+        final var newUserEntity = userMapper.map(userCreation);
+        newUserEntity.setPassword(passwordEncoder.encode(userCreation.password()));
         return userRepository.save(newUserEntity);
     }
 
     public UserEntity updateUser(final UserModification userModification, final String email) {
         final var userEntity = getUserByEmail(email);
         userEntity.setEmail(userModification.email());
-        userEntity.setPassword(passwordEncoder.encode(userModification.password()));
+        if (userModification.password() != null && !userModification.password().isEmpty()) {
```

Comment:
It's a standard operation to check null and empty. Such common things you can find in Apache library. In your case, you can use isNotEmpty https://www.educative.io/answers/what-is-stringutilsisnotempty-in-java